### PR TITLE
Feature#6 list layout

### PR DESCRIPTION
**Pull request by MiguelOcegueraM**
we created the mvc of the lists and we did the styles and design of the views 
scre1
scre2

#testing
follow the next steps and be sure to edit the database.yml file
you should have postgresql service active to test all

rails db:create
rails db:migrate
rails s (to start the server)
join to the localhost:3000/

### CODE REVIEW

**Review by OchoaHub — APPROVED** [review]

**Review by LPMendoza — APPROVED** [review]
Good job

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+.forms {
+    padding: 0 5%;
+}
+
+label {
+    color: $dark;
+}
+
+input[type="text"],
+textarea {
```
Comment (jaimelr) [inline app/assets/stylesheets/application.scss:193]: Be careful with using HTML tags to apply styles. This styles will apply to the all `textarea`s in the site and can get you in trouble if the project starts growing to the point where specific `textarea` inputs require particular styles. My suggestion is to use CSS classes named after the component + location of the element, for example: `.item-textarea`, `main-textarea`, etc.

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
+body {
```
Comment (jaimelr) [inline app/assets/stylesheets/scaffolds.scss:1]: You are modifying the `body` element across different files, which is not a good idea. This is a global element that can easily mess up the styles on your site. My suggestion is to group all changes related to global elements like `html` and `body` on a single file.

File: app/assets/stylesheets/scaffolds.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+#error_explanation {
+    width: 450px;
+    border: 2px solid red;
+    padding: 7px 7px 0;
+    margin-bottom: 20px;
+    background-color: #f0f0f0;
+}
+
+#error_explanation h2 {
```
Comment (jaimelr) [inline app/assets/stylesheets/scaffolds.scss:67]: Heads up: The use of ID selectors in CSS should be a kind of last-option. Prefer CSS classes.

This is important when your project begins to grow, there are priorities in CSS selectors:
High: `!important` <- Try not to use it at all.
           inline styles <- Like `<a style="background-color: #000"></a>`
           html tags <- Using the same HTML name in CSS e.g. `div { background-color: black }`. Global scope
           IDs <- `#error`, they are unique to an element and the styles can't be reused. They are also hard to override
Low:   CSS classes <- `.form` Easy to reuse, to name, and to override. No potential global side-effects.

File: app/views/todo_items/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<%=form_for([@todo_list, @todo_list.todo_items.build]) do |f| %>
+    <%=f.text_field :content, placeholder: "New Item" %>
+    <%= f.submit %>
```
Comment (jaimelr) [inline app/views/todo_items/_form.html.erb:3]: Check indentation.

File: app/views/todo_items/_todo_item.html.erb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  <div class="todo_item">
+   <p style="opacity: 0.4;"><strike><%= todo_item.content %></strike></p>
+  </div>
+  <div class="trash">
+   <%= link_to "Delete", todo_list_todo_item_path(@todo_list, todo_item.id), method: :delete, data: { confirm: "Are you sure?" } %>
+  </div>
+ <% else %>
+  <div class="complete">
+   <%= link_to complete_todo_list_todo_item_path(@todo_list, todo_item.id), method: :patch do %>
+    <i class="fa fa-circle-thin"></i>
```
Comment (jaimelr) [inline app/views/todo_items/_todo_item.html.erb:17]: Check indentation.

File: app/views/todo_lists/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<h1 class="todo_list_title">Editing To Do List</h1>
+<div class="forms">
+<%= render 'form', todo_list: @todo_list %>
+</div>
+<div class="links">
+<%= link_to 'Cancel', todo_lists_path %>
```
Comment (jaimelr) [inline app/views/todo_lists/edit.html.erb:6]: Check indentation.

File: app/views/todo_lists/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<h1 class="todo_list_title">New To Do List</h1>
+<div class="forms">
+<%= render 'form', todo_list: @todo_list %>
+</div>
+<div class="links">
+<%= link_to 'Back', todo_lists_path %>
```
Comment (jaimelr) [inline app/views/todo_lists/new.html.erb:6]: Check indentation.

**Review by jaimelr — COMMENTED** [review]
Overall looks good, just a couple of comments for you to consider either here or the next PR. 😉

File: app/views/todo_lists/new.html.erb (hunk 7 above)
Comment (MiguelOcegueraM) [reply app/views/todo_lists/new.html.erb:6]: Hi Jaime!! Thanks for all the feedback I appreciate it! Regards!

**Pull request merged**